client/src/App.js

```diff
@@ -2,39 +2,70 @@ import React, { useState, useEffect } from 'react';
 import './App.css';
 import Dashboard from './Components/Dashboard/Dashboard';
 import LoginPage from './Components/LoginPage/LoginPage';
+import PrivateRoute from './Components/PrivateRoute/PrivateRoute';
+import {
+  BrowserRouter as Switch,
+  Route,
+  BrowserRouter,
```

Comment:
I think we're importing `BrowserRouter` twice. Maybe what we wanted is something like the basic [example of React Router](https://reacttraining.com/react-router/web/example/basic)
In that case, we should modify:
```suggestion
  Switch,
  Route,
  BrowserRouter,
```